web/src/components/Testimonial/Testimonial.tsx

```diff
@@ -43,7 +42,7 @@ export const Testimonial: React.FC<TestimonialProps> = (
 		{
 			[styles[`testimonial--${kebabCaseVariantClassName}`]]: variant,
 		},
-		className
+		"testimonial"
```

Comment:
Pedantic but I'd put this as the first item in the array so its consistent with other components